src/EepromAT24C32.h

```diff
@@ -24,6 +24,9 @@ License along with Rtc.  If not, see
 <http://www.gnu.org/licenses/>.
 -------------------------------------------------------------------------*/
 
+#include <Arduino.h>
+#include "ThreeWire.h"
```

Comment:
The AT24C32 is not a three wire device, why are you trying to include the threewire object?